src/components/DisplayInput.js

```diff
@@ -84,20 +93,20 @@ const DisplayInput = React.createClass({
         </div>
 
         <Row>
```

Comment:
Do we really want the `status` message on a new row? I think it would be more intuitive to display it inside the `input` field on the far right. What do you think?